app/services/reports/querier.rb

```diff
@@ -17,12 +17,20 @@ def perform
       OrderDetailListTransformerFactory.instance(order_details).perform
     end
 
-    def order_details
+    def order_details(options = {})
```

Comment:
Or it seems like maybe we should be storing `options` from the initializer into `@options`?